packages/material-ui/src/TextField/TextField.js

```diff
@@ -197,7 +197,7 @@ TextField.propTypes = {
    */
   className: PropTypes.string,
   /**
-   * The default value of the `Input` element.
+   * The default value of the [`Input`](/api/input/) element.
```

Comment:
We have no guarantee about the component being rendered. It can be an Input, FilledInput or OutlinedInput. Either we use `The default value of the input element.` or ``The default value of the `InputBase` element.``

Reply:
Ah, makes sense. I just searched for capital `Input` and assumed it was the same class. I noticed you already fixed it in another commit. Thanks! 👍